datastream-server-restli/src/main/java/com/linkedin/datastream/server/dms/DatastreamResources.java

```diff
@@ -102,7 +102,7 @@ public class DatastreamResources extends CollectionResourceTemplate<String, Data
   // To support retries on the request timeouts
```

Comment:
Do we already have or should we have some test cases around these failure-causing / edge-case scenarios? 
1. Leader failure / kill
2. Follower failure / kill
3. Multiple stopping streams.
4. Stuck in the STOPPING state.

Reply:
Added a test case that simulates leader failover. I also added tests for the new force stop mechanism in DatastreamResources.

For follower failure/multiple stopping streams, I'm not sure what else to add. If there's a particular piece of code you want to have covered with tests (and which is not covered yet), let'd discuss.

Reply:
For follower failure, I was inferring the scenario where a follower fails to claim tokens, and then the leader cleans up for it. And for multiple stopping streams case, to see if more than one stream stopping is handled as expected.